config.py

```diff
@@ -0,0 +1,12 @@
+UPSTREAMFIRST_URL = 'https://upstreamfirst.fedorainfracloud.org/'
+DIST_GIT_URL = 'https://src.fedoraproject.org/'
+
+FILE_UPLOAD_2_WIKI = 'upl2wiki.txt'
+FILE_PACKAGES = 'upstreamfirst_packages.txt'
```

Comment:
No such file as 'upstreamfirst_packages.txt'

Reply:
Agree. This file will be generated automatically. I save packages statistic in the file instead of keeping it in RAM